spec/spec.md

```diff
@@ -1275,6 +1275,31 @@ format-related rules above:
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
+    "filter": {
+      "type": "object",
+      "properties": {
+        "type": { "type": "string" },
+        "format": { "type": "string" },
+        "pattern": { "type": "string" },
+        "minimum": { "type": ["number", "string"] },
+        "minLength": { "type": "integer" },
+        "maxLength": { "type": "integer" },
+        "exclusiveMinimum": { "type": ["number", "string"] },
+        "exclusiveMaximum": { "type": ["number", "string"] },
+        "maximum": { "type": ["number", "string"] },
+        "const": { "type": ["number", "string"] },
+        "enum": { 
+          "type": "array",
+          "items": { "type": ["number", "string"] }
+        },
+        "not": {
```

Comment:
Should the value of `not` be an array or an object? (all of my examples are objects, I think).

Reply:
oops, updated.